src/index.tsx

```diff
@@ -59,13 +60,22 @@ const wallets = [
 
 ReactDOM.render(
   <React.StrictMode>
-    <QueryClientProvider client={queryClient}>
-      <AptosWalletAdapterProvider plugins={wallets} autoConnect={true}>
-        <BrowserRouter>
-          <ExplorerRoutes />
-        </BrowserRouter>
-      </AptosWalletAdapterProvider>
-    </QueryClientProvider>
+    <StatsigProvider
+      sdkKey={process.env.REACT_APP_STATSIG_SDK_KEY || ""}
+      waitForInitialization={true}
+      options={{
+        environment: {tier: "production"},
+      }}
+      user={{}}
```

Comment:
could add in the future once we launch wallet connection in mainnet but we prob wanna use wallet address, etc as the user identification here?

Reply:
yes for features with wallet connection we'll need to include user. but for most of explorer features like this demo we don't need it. let's add it upon needed.